### PR TITLE
Add benchmarks for queue

### PR DESCRIPTION
**Pull request by howardjohn**
**Please provide a description of this PR:**

### CODE REVIEW

File: pkg/queue/instance.go [hunk 1]
```diff
@@ -22,8 +22,51 @@ import (
...
+		"pilot_queue_active",
+		"Total number of items in progress.",
+		monitoring.WithLabels(idTag),
+	)
+
+	qerrors = monitoring.NewSum(
+		"pilot_queue_error_total",
+		"Total number of tasks errored and retried.",
+		monitoring.WithLabels(idTag),
+	)
```
[the remark this answers is not in the document]
Comment (howardjohn) [reply pkg/queue/instance.go:59]: Oop,s meant to only include the benchmark. Accidentally rebased on Rama's change since I was testing it

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by GregHanson** [comment]
@howardjohn @hzxuzhonghu should this get cherrypicked to 1.15 branch?

**Comment by howardjohn** [comment]
Doesn't matter

**Pull request merged**